entrypoint.sh

```diff
@@ -5,6 +5,7 @@ set -o errexit -o pipefail -o nounset
 echo '::group::Creating builder user'
 useradd --create-home --shell /bin/bash builder
 passwd --delete builder
+echo "builder  ALL=(root) NOPASSWD:ALL" > /etc/sudoers.d/builder
```

Comment:
Why use overwrite (`>`) instead of just append (`>>`)?

Reply:
I am creating a new file in the `/etc/sudoers.d/` folder, which is supposed not to exist.
In this way I avoid appending to the "main" `/etc/sudoers` configuration file.

Reply:
```suggestion
mkdir -p /etc/sudoers.d/
echo "builder  ALL=(root) NOPASSWD:ALL" > /etc/sudoers.d/builder
```